frontend/src/components/CreateCollectionModal/components/Content/components/LinkInput/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { IconNames } from "@blueprintjs/icons";
+import { debounce } from "lodash-es";
+import React, { FC } from "react";
+import Input from "src/components/common/Form/Input";
+import { GRAY } from "src/components/common/theme";
+import { DEBOUNCE_TIME_MS } from "src/components/CreateCollectionModal/components/Content/common/constants";
+import { IconWrapper, StyledButton, Wrapper } from "./style";
+
+export type LinkValue = {
+  id: number;
+  value: string;
+  isValid: boolean;
+  index: number;
+};
+
+export enum TYPES {
+  DOI = "DOI",
+  RAW_DATA = "RAW_DATA",
+  PROTOCOL = "PROTOCOL",
+  LAB_WEBSITE = "LAB_WEBSITE",
+  OTHER = "OTHER",
+}
```

Comment:
This enum is a duplicate of the one on entities

Reply:
Yes! It's fixed in this [PR](https://github.com/chanzuckerberg/corpora-data-portal/pull/689) :D Thanks for the callout!

Reply:
Somehow Github is not reflecting the fix from your PR, but I just verified that the TYPE is now consolidated!